queries.sql

```diff
@@ -8,3 +8,29 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg >= 10.50;
 SELECT * FROM animals WHERE neutered = true; 
 SELECT * From animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+UPDATE animals
+SET species = 'digimon' 
+WHERE name LIKE '%mon%';
+
+UPDATE animals
+SET species = 'pokemon' 
+WHERE name NOT LIKE '%mon%';
```

Comment:
This query must be performed inside a Transaction. Kindly add `BEGIN` before the update and `COMMIT` after the update to end the transaction.